Hookies/Game/Engine/GameEngine.swift

```diff
@@ -479,6 +515,29 @@ class GameEngine {
         }
         return nil
     }
+
+    // MARK: - Bots
+
+    private func addBot(sprite: SpriteComponent, botType: BotType) {
+        let botEntity = BotEntity(botType: botType)
+        guard let botComponent = botEntity.get(BotComponent.self) else {
+            return
+        }
+        guard let botSystem = self.botSystem else {
+            return
+        }
+        botSystem.add(spriteComponent: sprite, botComponent: botComponent)
+        delegate?.addLocalPlayer(with: sprite.node)
+    }
+
+    func initialiseBotSystem(_ players: [Player]) {
```

Comment:
Do we need another function to initialise `BotSystem`? We can do it within the `init` of `gameEngine` since it does take in any parameters. Within the other player device, they also won't render a `bot`

Reply:
Yup, ok I'll move the check to the game scene.